docs/api.rst

```diff
@@ -0,0 +1,537 @@
+#################
+API Documentation
+#################
+
+
+GET /
+-----
```

Comment:
We probably should use `====` for first level titles, and `----` for second level ones.